changelogs/unreleased/gh-6986-introduce-constraint-interface.md

```diff
@@ -0,0 +1,4 @@
+## feature/box
+
+* The space object in Lua now has an interface to create or drop foreign keys
+  and constraints (gh-6986).
```

Comment:
We shouldn't modify or extend the existing API without an approval from the PM team. Anyway, I don't understand *why* you need these new Lua methods. The idea was to introduce the C API that could be used by SQL. Ideally, we should rewrite the *existing* Lua API using it if possible, not introduce *new* API.

Reply:
I convert this PR to draft for now.